.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,32 @@
+name: Documentation
+
+on:
+  push:
+    branches: [main, master]
+
+jobs:
+  make-pages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: select python version
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .
+          python -m pip install sphinx
+          python -m pip install sphinx_rtd_theme
+      - name: build documentation
+        run: |
+          cd docs
+          make html
```

Comment:
This will fail because of a dependency on `plantuml`. It would probably be easiest to remove the dependency. The chart it creates is not very valuable.